src/components/Lightbox.tsx

```diff
@@ -212,17 +204,8 @@ function Lightbox({
                                 resizeMode="contain"
                                 style={fallbackSize}
                                 isAuthTokenRequired={isAuthTokenRequired}
-                                onLoadEnd={() => setFallbackLoaded(true)}
-                                onLoad={(e: ImageOnLoadEvent) => {
-                                    const width = e.nativeEvent.width * PixelRatio.get();
-                                    const height = e.nativeEvent.height * PixelRatio.get();
-
-                                    if (isLightboxLoaded) {
-                                        return;
-                                    }
-
-                                    setImageDimensions({...imageDimensions, fallbackSize: {width, height}});
-                                }}
+                                onLoad={updateContentSize}
```

Comment:
If we render both the image and the fallback, wouldn't the fallback override the image size? Shouldn't we re-add the `isLightboxImageLoaded` early return check?

Reply:
Actually, it doesn't matter whether the `contentSize` is set by the lightbox image vs. the fallback image, as long as it's being set. It will be the same from both `onLoad` events. But you're right, we don't have to set it again. I added a early return...